Source/Scene/Cesium3DTileset.js

```diff
@@ -349,6 +329,36 @@ define([
         when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
     }
 
+    function selectTileWithTilesetContent(tiles3D, selectedTiles, tile, fullyVisible, frameState, replace) {
+        // 1) If its children are not loaded, load the subtree it points to and then select its root child
+        // 2) If its children are already loaded, select its (root) child since the geometric error of it is
+        //    same as this tile's
+        var childrenLength = tile.children.length;
+        var contentUrl = tile._header.content.url;
+
+        // If the subtree has already been added and child
+        // content requested, select the child and continue
+        if ((childrenLength > 0) && (tile.numberOfChildrenWithoutContent === 0)) {
+            selectTile(selectedTiles, tile.children[0], fullyVisible, frameState);
+            return;
+        } else if (replace) {
+            // Otherwise, select the parent tile, to avoid showing an empty space
+            // Note that tile.parent must always be defined here --
+            // tile should never be the root of the tileset AND
+            // contain another tileset
+            selectTile(selectedTiles, tile.parent, fullyVisible, frameState);
+        }
+
+        // Request the tile's tileset.
+        if (childrenLength === 0 && !tile.tilesetLoading) {
```

Comment:
Ah, I see, this is a bit too implicit.  Two questions:

* [ ] Same spec question as before: should the spec should say that when `content.type` is a tileset, the tile should not have any children?  Maybe this limitation isn't needed since it should be fairly easy to implement concatenating the children with the other tileset's root.
* [ ] What do you think about replacing this check and `tilesetLoading` with an enum, something like:
```javascript
    var TilesetContentStatus = {
        UNLOADED : 0,
        LOADING : 1,
        READY : 2
    };
```
This is derived from `Cesium3DTileContentState`.